### PR TITLE
Wallet user update

### PR DESCRIPTION
**Pull request by waveform06**
Post solana phrasing with updates and added info on how to identify migrated wallet.

### CODE REVIEW

**Comment by waveform06** [comment]
Actually helium hotspot app needs to be 2.0.8 or higher

File: docs/architecture/solana/migration/wallet-user.mdx [hunk 1]
```diff
@@ -9,24 +9,23 @@ slug: /solana/migration/wallet-user
...
+wallet is migrated. Token balances are visible on the account page. You can also create or import multiple
 accounts to a maximum of 10 on the Helium Wallet app.
 
 Existing HNT and MOBILE tokens in your wallets will remain unchanged by this migration.
 
 ## Migration Utility
 
 You can migrate a wallet using the widget below by entering a Helium or Solana wallet address and
-clicking the "Seed Wallet" buttton.
+clicking the "Seed Wallet" buttton. Wallets seeded/migrated will have 0.00139088 SOL deposited in
```
Comment (jthiller) [inline docs/architecture/solana/migration/wallet-user.mdx]: typo, "buttton"

File: docs/architecture/solana/migration/wallet-user.mdx [hunk 2]
```diff
@@ -38,13 +37,14 @@ import { MigrateWallet } from '@theme/MigrateWallet'
 
 The Helium Hotspot App is no longer used to manage Tokens or perform Wallet activities and instead
 is only used to manage Original Helium Hotspots and FreedomFi Hotspots.
+Hotspots are now managed with Maker Apps or the Helium Wallet.
```
Comment (jthiller) [inline docs/architecture/solana/migration/wallet-user.mdx]: Should downplay maker apps as we're working to bring these features back to Helium Wallet.

File: docs/architecture/solana/migration/wallet-user.mdx [hunk 3]
```diff
@@ -38,13 +37,14 @@ import { MigrateWallet } from '@theme/MigrateWallet'
...
 
 :::
 
 Helium Hotspot app users must migrate to the Helium Wallet App following these steps.
 
-To start, ensure your Helium Hotspot app is updated to version **5.0.0** or later and that you have
-downloaded the latest Helium Wallet app version **2.0.0** or higher.
+To start, ensure your Helium Hotspot app is updated to at least version **5.0.0** or later and that 
+the Helium Wallet app version **2.0.0** or later. The latest versions of each app are recommended.
```
Comment (jthiller) [inline docs/architecture/solana/migration/wallet-user.mdx]: "The latest versions of each app are recommended." not needed

**Review by jthiller — CHANGES_REQUESTED** [review]

**Review by jthiller — APPROVED** [review]

**Pull request merged**